chapters/data-processing.tex

```diff
@@ -0,0 +1,783 @@
+%------------------------------------------------
+
+\begin{fullwidth}
+
+% Chapter overview
+The tasks described in this chapter have many different names.
+Some say data cleaning, others data munging, or maybe data wrangling.
+All mean transforming raw data into a more convenient format for your intended use.
+Although original data typically requires more extensive data cleaning than secondary data,
+it is broadly agreed that this is, on average, the most time-consuming step in a project's data work \citep{dasu2003exploratory}.
+This chapter describes the different tasks involved in the cleaning process,
+and outlines the workflow our team developed to make the process efficient and transparent.
+
+% What is data cleaning
+This chapter describes all tasks needed to make newly acquired data ready for analysis.
+The first section will guide on making your data \textit{tidy},
+so the relationship between rows and columns in a data set is well-defined.
+This includes removing duplicated observations.
+The second section describes quality assurance checks, which are necessary to verify the accuracy of your data
+After that, identifying information is typically no longer needed,
+the third section outlines steps to remove direct identifiers.
+Finally, the last section discusses how to explore each variable in your dataset and 
+make sure that the dataset is as well documented and easy to use as possible.
+
+
+\end{fullwidth}
+
+%------------------------------------------------
+
+\subsection{Before you get started}
```

Comment:
This is the only chapter that starts with a subsection. Not incorrect but I thought it broke the flow a little. I think you can have `\section{Identifying the identifier(s)}` and then say "Before you get started".